### DISCUSSION TITLE
Installing via installer vs via Microsoft Store

### DISCUSSION THREAD

**Discussion by ksze** [PowerShell/PowerShell#16426] [Q&A]
On Windows 10/11, is there any practical difference between installing PowerShell from, say, the PowerShell-7.x.x-win-x64.msi vs installing from Microsoft Store?

I suppose that PowerShell from the Microsoft Store would get auto-updated. Any other difference?

**Comment by Kaushikpandav** [comment]
One time uninstall and again install And if that doesn't happen, it's time to dump her and move on Google and and upgrade that set 👍

**Reply by lukina4real** [reply]
Is that advised?

**Comment by iSazonov** [comment]
pwsh from Store is a sandboxed application and it can have (unexpected) limitations to access OS resources.

**Comment by doctordns** [comment]
For production use, I recommend using the Install-PowerShell script from the PowerShell github repo.

This works well for installing RTM, Preview and Build of the day.
